playground/backend/cmd/server/server.go

```diff
@@ -107,8 +108,13 @@ func setupCache(ctx context.Context, appEnv environment.ApplicationEnvs) (cache.
 }
 
 func main() {
+	os.Setenv("PROTOCOL_TYPE", "TCP")
```

Comment:
It seems that we do not need this one.

Reply:
Sorry it was draft version, needs it for testing